### PR TITLE
EWS: user extension

### PR DESCRIPTION
**Pull request by kkrampa**
@czue  I made this feature according to Mike's suggestion to put button in user edit page. I think in this solution we are touching core code in minimal way. Let me know if you have better idea how we can solve this.

### CODE REVIEW

**Comment by czue** [comment]
I can't think of anything that's much better that doesn't seem like it would involve a lot of work. pinging @esoergel and @biyeun who I think own this part of the codebase and might have tips.

the issue is that this custom project needs to add a button to the web user page. this seems like a very explicit (if also very custom) way to do that. do you have any other ideas on anything better?

File: corehq/apps/users/views/__init__.py [hunk 1]
```diff
@@ -267,6 +268,7 @@ def main_context(self):
     def page_context(self):
         ctx = {
             'form_uneditable': BaseUserInfoForm(),
+            'ews_extension': EWS_WEB_USER_EXTENSION.enabled(self.domain)
```
Comment (esoergel) [inline corehq/apps/users/views/__init__.py]: You can avoid passing additional context to the template by using the `toggle_enabled` template filter:
```jinja
{% if request|toggle_enabled:"EWS_WEB_USER_EXTENSION" %}
```

**Comment by esoergel** [comment]
Aside from that one suggestion, I can't think of a better alternative.

File: corehq/apps/users/views/__init__.py (hunk 1 above)
Comment (biyeun) [inline corehq/apps/users/views/__init__.py]: seconded

File: corehq/apps/users/views/__init__.py [hunk 2]
```diff
@@ -17,6 +17,7 @@
 )
 from corehq.apps.users.decorators import require_can_edit_web_users, require_permission_to_edit_user
 from corehq.elastic import ADD_TO_ES_FILTER, es_query, ES_URLS
+from corehq.toggles import EWS_WEB_USER_EXTENSION
```
Comment (dimagimon) [inline corehq/apps/users/views/__init__.py]: F401 'EWS_WEB_USER_EXTENSION' imported but unused

**Comment by kkrampa** [comment]
Updated

**Pull request merged**